src/components/detailsPage.js

```diff
@@ -0,0 +1,163 @@
+                alt="circled-left-2"
+              />
+            </div>
+            <div className="one">
+              <p>Feels like:</p>
+              <p>
+                {current.feelslike}
+                {' '}
+                °C
+              </p>
+              <img
+                className="arrow-two"
+                src="https://img.icons8.com/ios/50/ffffff/circled-right-2.png"
+                alt="circled-left-2"
+              />
+            </div>
+            <div className="two">
+              <p>Wind speed:</p>
+              <p>
+                {current.wind_speed}
+                {' '}
+                km/h
+              </p>
+              <img
+                className="arrow-two"
+                src="https://img.icons8.com/ios/50/ffffff/circled-right-2.png"
+                alt="circled-left-2"
+              />
+            </div>
+            <div className="one">
+              <p>Wind degree:</p>
+              <p>
+                {current.wind_degree}
+                °
+              </p>
+              <img
+                className="arrow-two"
+                src="https://img.icons8.com/ios/50/ffffff/circled-right-2.png"
+                alt="circled-left-2"
+              />
+            </div>
+            <div className="two">
+              <p>Preassure:</p>
+              <p>
+                {current.pressure}
+                {' '}
+                mb
+              </p>
+              <img
+                className="arrow-two"
+                src="https://img.icons8.com/ios/50/ffffff/circled-right-2.png"
+                alt="circled-left-2"
+              />
+            </div>
+            <div className="one">
+              <p>Humidity:</p>
+              <p>
+                {current.humidity}
+                %
+              </p>
+              <img
+                className="arrow-two"
+                src="https://img.icons8.com/ios/50/ffffff/circled-right-2.png"
+                alt="circled-left-2"
+              />
+            </div>
+            <div className="two">
+              <p>Cloud cover:</p>
+              <p>
+                {current.cloudcover}
+                %
+              </p>
+              <img
+                className="arrow-two"
+                src="https://img.icons8.com/ios/50/ffffff/circled-right-2.png"
+                alt="circled-left-2"
+              />
+            </div>
+            <div className="one">
+              <p>Visibility:</p>
+              <p>
+                {current.visibility}
+                {' '}
+                km
+              </p>
+              <img
+                className="arrow-two"
+                src="https://img.icons8.com/ios/50/ffffff/circled-right-2.png"
+                alt="circled-left-2"
+              />
+            </div>
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+export default DetailsPage;
```

Comment:
- [ ] When I visit the details page of a country, it shows the page but I notice that when I reload the page, it doesn't show anything. The gif below illustrates my point better 🖱️ :

![chrome-capture-2023-8-6](https://github.com/agneta1991/metrics-webapp/assets/77106962/9f7bb7c8-8f21-4aba-b033-b73a8b5fd7dd)

These are the errors I get in the console of the browser when I reload the page:

![image](https://github.com/agneta1991/metrics-webapp/assets/77106962/e00e56eb-d7d9-402b-8c74-d2fa6b7c95f9)

Kindly fix the error and ensure that the page functions properly. 👍